cmd/lotus-shed/balances.go

```diff
@@ -170,7 +171,9 @@ var chainBalanceStateCmd = &cli.Command{
 
 		sm := stmgr.NewStateManager(cs)
 
-		tree, err := state.LoadStateTree(cst, sroot)
+		// NETUPGRADE: FIXME.
+		// Options: (a) encode the version in the chain or (b) pass a flag.
+		tree, err := state.LoadStateTree(cst, sroot, network.Version0)
```

Comment:
All of the commands that simply take state roots are going to be a pain. Can we just make them take chain heights? or tipset keys?

Reply:
oof.. Those commands generally have a very good reason to use stateroots. I'd make them take the network version from the current chain head, and allow overriding with some flag

Reply:
Ok. We've changed the state root to include a version.